src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,11 +1,15 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int showelType;
```

Comment:
add getter/setter for field, now it's impossible to access it (same with other fields in models)